DER/DerivativesContracts/CurrencyContracts.rdf

```diff
@@ -86,7 +87,12 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fbc-fi-fi;hasUnderlier"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-rtd-rtd;ForeignExchangeRateObservable"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-ind-fx-fx;QuotedExchangeRate"/>
```

Comment:
There's a bigger issue I noticed here, related to the whole definition of RateObservable - it should be something you observe over time not a specific value. However QuotedExchangeRate is defined as "exchange rate quoted at a specific point in time" and indeed that's reflected in its superclass of MarketRate.